.gitignore

```diff
@@ -14,3 +14,6 @@ tarballs
 # Where do these files come from?  They're not readable.
 # For instance, .#Help.page
 .#*
+.stack-work/
+stack.yaml.lock
+*.swp
```

Comment:
Wouldn't these patterns be better off in Stack users' global gitignore file?

What's `*.swp`?

Reply:
It's for vim backup files. When you open a file in vim it creates a swp file so that if it crashes you don't lose the information.

Reply:
I'm inclined to remove rather than add more ignored files in the project, relying more on users global gitignore files.

Reply:
Sounds good. I can close this.